data/autoyast_hanaperf/agama-config.jsonnet

```diff
@@ -19,7 +19,7 @@
   storage: {
     drives: [
       {
-        search: '/dev/disk/by-id/scsi-SATA_DELLBOSS_VD_b68e1f8449390010',
+        search: '/dev/disk/by-id/{{OSDISK}}',
```

Comment:
Where is this set? Changing this makes the setting mandatory, right?

Reply:
The variable is set in machine or worker configuration, it inherited from autoyast template:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/3bd20520a8e0cd051268c2503845e349c435e99b/data/autoyast_hanaperf/sle15.xml.ep#L128